metadata-protobuf/proto/Bounty.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto2";
+
+message BountyMetadata {
+  optional string title = 1; // Bounty title
+  optional string description = 2; // Bounty description
+  optional uint32 discussionThread = 3; // Id of the forum thread used to discuss the bounty
```

Comment:
I think `discussionThread` should be of type `uint64` because threads are defined as a 64-bit number in runtime. https://github.com/Joystream/joystream/blob/ed663cccf7f0a3d2cef2abdbf7e416233911ec2a/runtime/src/primitives.rs#L85

Reply:
Thanks I missed that